docs/coordinate_systems.md

```diff
@@ -0,0 +1,177 @@
+
+![Wormole harvesting](images/tensix_harvested_rows.png)
+
+Note that there is no limitation on which specific rows can be harvested.
+
+Harvesting of non-tensix cores (DRAM, PCIe, ARC, Ethernet) is not supported on Wormhole.
+
+### Blackhole harvesting
+
+On Blackhole, harvesting of tensix columns is supported. That means that on the tensix grid (14x10), there will always be 10 rows of tensix cores, but number of columns may decrease. In practice, Blackhole chips have (TODO: how many columns) columns harvested. Example for two harvested columns is in the image below.
+
+(TODO: attach image of harvested columns)
+
+Note that there is no limitation on which specific columns can be harvested.
+
+(TODO: dram harvesting)
+
+Harvesting of other cores (PCIe, ARC, Ethernet) is not supported on Blackhole.
+
+# Coordinate systems
+
+In the following section, coordinate systems that can be used to access cores are going to be described. For each coordinate system it will be described how does harvesting affect access to different cores when using certain coordinate system, as well as the API support through UMD for this coordinate system.
+
+To keep everything consistent, example of 8x10 grid, which represents Wormhole chip, is going to be used in each image in this section.
+
+In order to illustrate harvesting effect on coordinate systems, example with two harvested rows of Tensix cores (image below) is going to be used
+
+## Logical coordinates
+
+This coordinate system is mostly used to reference Tensix cores, since this cores are most frequently accessed. This coordinate system hides the details of physical coordinates and allows upper layers of the stack to access Tensix endpoints through a set of traditional Cartesian Coordinates. This coordinate systems has very simple indexing, it starts from `0-0` and ends at `(X-1)-(Y-1)` where X and Y is number of cores on x-axis and y-axis, respectively. Example on logical coordinate indexing is one the image below
+
+![Tensix logical coordinates](images/tensix_logical_coordinates.png)
+
+Logical coordinates are used in a same way for Grayskull, Wormhole and Blackhole, only difference being in grid size of tensix cores.
+
+### Harvesting effect on logical coordinates
+
+Using harvesting example, the effect on logical coordinates for two harvested configuration is on the image below
+
+![Tensix logical coordinates harvested](images/tensix_logical_coordinates_harvested.png)
+
+Note that range on X axis stays the same (no harvested columns), but the range on Y axis is smaller by two (number of harvested rows).
+
+## Physical Coordinates
+
+These are the NOC coordinates that the hardware understands, there are two distinct variations for NOC0 and NOC1. In hardware, each node is given an ID (which is different for each NOC), represented as x-y pair, which can be used to identify this node. In the SOC descriptor, physical coordinates are specified for NOC0.
```

Comment:
Is there a reason we use the word physical for these?  I think it'd be clearer to call them NOC coordinates.

The physical layout of the chip (if you are looking down at it) is _not_ consistent with what we have been calling "physical" coordinates.